src/mapping/impl/SphericalVertexCluster.hpp

```diff
@@ -56,14 +56,15 @@ class SphericalVertexCluster {
                          double                  radius,
                          RADIAL_BASIS_FUNCTION_T function,
                          Polynomial              polynomial,
+                         bool                    crossValidation,
                          mesh::PtrMesh           inputMesh,
                          mesh::PtrMesh           outputMesh);
 
   /// Evaluates a conservative mapping and agglomerates the result in the given output data
   void mapConservative(const time::Sample &inData, Eigen::VectorXd &outData) const;
 
   /// Evaluates a consistent mapping and agglomerates the result in the given output data
-  void mapConsistent(const time::Sample &inData, Eigen::VectorXd &outData) const;
+  std::array<double, 3> mapConsistent(const time::Sample &inData, Eigen::VectorXd &outData) const;
```

Comment:
The "3" is the number of components, right?
So, for scalar data the last two components will remain unused?